lib/matplotlib/animation.py

```diff
@@ -1598,10 +1598,14 @@ def init_func() -> iterable_of_artists
        blitting any animated artists will be drawn according to their zorder.
        However, they will be drawn on top of any previous artists, regardless
        of their zorder.  Defaults to *False*.
+
+    cache_frame_data : bool, optional
+       Controls whether frame data is cached. Defaults to *True*.
+       Disabling cache might be helpful when frames contain large objects.
     """
 
     def __init__(self, fig, func, frames=None, init_func=None, fargs=None,
-                 save_count=None, **kwargs):
+                 save_count=None, *args, cache_frame_data=True, **kwargs):
```

Comment:
should be `save_count=None, *, cache_frame_data=True, **kwargs`

Reply:
@anntzer 
Thanks for your feedback.
I've looked up `single star` syntax in [PEP 3102](https://www.python.org/dev/peps/pep-3102/) and it makes sense!